shared_model/builders/protobuf/queries.hpp

```diff
@@ -0,0 +1,142 @@
+          : query_(o.query_) {}
+
+     public:
+      TemplateQueryBuilder() = default;
+
+      NextBuilder<CreatedTime> createdTime(uint64_t created_time) {
+        query_.mutable_payload()->set_created_time(created_time);
+        return *this;
+      }
+
+      NextBuilder<CreatorAccountId> creatorAccountId(
+          const interface::types::AccountIdType &creator_account_id) {
+        query_.mutable_payload()->set_creator_account_id(creator_account_id);
+        return *this;
+      }
+
+      NextBuilder<QueryField> setGetAccount(
+          const interface::types::AccountIdType &accound_id) {
+        auto query = query_.mutable_payload()->mutable_get_account();
+        query->set_account_id(accound_id);
+        return *this;
+      }
+
+      NextBuilder<QueryField> setGetSignatories(
+          const interface::types::AccountIdType &accound_id) {
+        auto query =
+            query_.mutable_payload()->mutable_get_account_signatories();
+        query->set_account_id(accound_id);
+        return *this;
+      }
+
+      NextBuilder<QueryField> setGetAccountTransactions(
+          const interface::types::AccountIdType &accound_id) {
+        auto query =
+            query_.mutable_payload()->mutable_get_account_transactions();
+        query->set_account_id(accound_id);
+        return *this;
+      }
+
+      NextBuilder<QueryField> setGetAccountAssetTransactions(
+          const interface::types::AccountIdType &accound_id,
+          const interface::types::AssetIdType &asset_id) {
+        auto query =
+            query_.mutable_payload()->mutable_get_account_asset_transactions();
+        query->set_account_id(accound_id);
+        query->set_asset_id(asset_id);
+        return *this;
+      }
+
+      NextBuilder<QueryField> setGetAccountAssets(
+          const interface::types::AccountIdType &accound_id,
+          const interface::types::AssetIdType &asset_id) {
+        auto query = query_.mutable_payload()->mutable_get_account_assets();
+        query->set_account_id(accound_id);
+        query->set_asset_id(asset_id);
+        return *this;
+      }
+
+      NextBuilder<QueryField> setGetRoles(
+          const interface::types::AccountIdType &accound_id,
+          const interface::types::AssetIdType &asset_id) {
+        query_.mutable_payload()->mutable_get_roles();
+        return *this;
+      }
+
+      NextBuilder<QueryField> setGetAssetInfo(
+          const interface::types::AssetIdType &asset_id) {
+        auto query = query_.mutable_payload()->mutable_get_asset_info();
+        query->set_asset_id(asset_id);
+        return *this;
+      }
+
+      NextBuilder<QueryField> setGetRolePermissions(
+          const interface::types::RoleIdType &role_id) {
+        auto query = query_.mutable_payload()->mutable_get_role_permissions();
+        query->set_role_id(role_id);
+        return *this;
+      }
+
+      NextBuilder<QueryCounter> queryCounter(uint64_t query_counter) {
```

Comment:
Style issue: group this function with created_time and creator_account_id setters.

Reply:
This functions follow the order fields in the proto message